scooter/src/main.rs

```diff
@@ -660,4 +666,25 @@ mod tests {
         let err = result.unwrap_err().to_string();
         assert!(err.contains("not a directory"));
     }
+
+    #[test]
+    fn test_editor_command_override() {
+        let args = Args {
+            editor_command: Some("vim %file +%line".to_string()),
```

Comment:
Small thing but the test runner in CI will probably default to `vim` on some platforms anyway. Maybe could swap `vim` out with something that is clearly not the default e.g. `my_editor_override`?

```suggestion
            editor_command: Some("my_editor_override %file +%line".to_string()),
```